docs/user/openstack/add_worker_nodes_by_hand.md

```diff
@@ -0,0 +1,68 @@
+#Adding workers nodes by hand
+The prefered method of adding worker nodes to an OpenShift cluster is
+to use the a worker machineset and machine api operator as described in
+https://github.com/openshift/machine-api-operator.
+
+It is also possible to add a worker through other tools like ansible. For example, 
+the workers may be added as described in [Installing OpenShift on OpenStack User-Provisioned Infrastructure][1];
+or when scaling up the workers using [Openshift-ansible][2]
+
+When doing so, there are a few important points that should be taken into account. 
+
+##Apply Necessary Security Groups To Ports
+When adding a new worker node, the necessary security groups need to be applied so the node is 
+allowed access the cluster API endpoints and services.   
+
+If the node is directly attached the machineNetwork,  the pre-existing worker security groups maybe 
+applied to the port attaching the node. The security rules may require little to no modifications.  
+
+If the nodes being added is attached to a different network than the machineNetwork,
+then new rules will have to be configured because the rules deployed by default on a cluster
+match on both destination and source networks.
```

Comment:
How will the security group rules need to be changed in this use case?

Reply:
Its dificult to predict how the security groups need to be modified because this configuration (different network than machineNetwork), pretty much is everything else. 
not sure we can give more detail on how to do it without leaving out 90% of cases.